src/containers/org/OrgRoleContainer.js

```diff
@@ -55,6 +61,9 @@ const OrgRoleContainer = ({
   rolesRoute :string;
 |}) => {
 
+  const [filterByPermissionTypes, setFilterByPermissionTypes] = useState([]);
+  const [filterByQuery, setFilterByQuery] = useState('');
```

Comment:
Perhaps no change required here, but `filterByPermissionTypes` and `filterByQuery` as names of the stateful values sound rather awkward

Reply:
I went with this basically because I wanted them to be together when sorting. I considered `permissionTypesToFilterBy` and `queryToFilterBy`, but those are rather lengthy and won't necessarily be together in props. I also considered `filterQuery` and `filterPermissionTypes`, but I like `filterBy` better.

Reply:
well, `filterByPermissionTypes` with no additional context would most likely be assumed to be the name of a function, not an array as is the case here. The same could be said about `filterByQuery`.